docs/eigenda/deployed-contracts.md

```diff
@@ -0,0 +1,25 @@
+|--------------------------------|------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|---------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| RegistryCoordinator            | [`BLSRegistryCoordinatorWithIndices.sol`](https://github.com/Layr-Labs/eigenlayer-middleware/blob/v0.1.0-m2-goerli/src/BLSRegistryCoordinatorWithIndices.sol) | [`0x7661dA87296c0A245d11ADeD7f48265463bc3eC6`](https://goerli.etherscan.io/address/0x7661dA87296c0A245d11ADeD7f48265463bc3eC6) | [`0x7dfE0C25483E420eccD04eab8Be44c75a0CE82B5`](https://goerli.etherscan.io/address/0x7dfE0C25483E420eccD04eab8Be44c75a0CE82B5) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
+| StakeRegistry                  | [`StakeRegistry.sol`](https://github.com/Layr-Labs/eigenlayer-middleware/blob/v0.1.0-m2-goerli/src/StakeRegistry.sol)                         | [`0x9949f23AD76A8B6E9E090621B10D7d5fC93eCfCF`](https://goerli.etherscan.io/address/0x9949f23AD76A8B6E9E090621B10D7d5fC93eCfCF) | [`0xFf76a8549A56F880EdCD5b911F87389b03aF9536`](https://goerli.etherscan.io/address/0xFf76a8549A56F880EdCD5b911F87389b03aF9536) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
+| IndexRegistry                  | [`IndexRegistry.sol`](https://github.com/Layr-Labs/eigenlayer-middleware/blob/v0.1.0-m2-goerli/src/IndexRegistry.sol)                         | [`0x36AA7406E4a724e5ba3e58C9C2703B4Bf532FE3F`](https://goerli.etherscan.io/address/0x36AA7406E4a724e5ba3e58C9C2703B4Bf532FE3F) | [`0x317654870F914473642838ACA30f7ddc64f68Beb`](https://goerli.etherscan.io/address/0x317654870F914473642838ACA30f7ddc64f68Beb) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
+| BLSApkRegistry                 | [`BLSPubkeyRegistry.sol`](https://github.com/Layr-Labs/eigenlayer-middleware/blob/v0.1.0-m2-goerli/src/BLSPubkeyRegistry.sol)                   | [`0xb406d8e5F4BA1430D57c8231cc872C2c369E1F5F`](https://goerli.etherscan.io/address/0xb406d8e5F4BA1430D57c8231cc872C2c369E1F5F) | [`0x7f57f298b7270B1D4C8964E33eB6eEfF31aD2063`](https://goerli.etherscan.io/address/0x7f57f298b7270B1D4C8964E33eB6eEfF31aD2063) | Proxy: [OpenZeppelin TUP@4.7.1](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.7.1/contracts/proxy/transparent/TransparentUpgradeableProxy.sol) |
```

Comment:
This should be the right solidity file for BLSApkRegistry: https://github.com/Layr-Labs/eigenlayer-middleware/blob/m2-mainnet-fixes/src/BLSApkRegistry.sol